corehq/apps/callcenter/fixturegenerators.py

```diff
@@ -13,17 +15,26 @@ def should_sync(domain, last_sync, utcnow=None):
     if not last_sync or not last_sync.date:
         return True
 
+    # utcnow only used in tests to mock other times
+    utcnow = utcnow or datetime.utcnow()
+
     try:
-        timezone = pytz.timezone(domain.default_timezone)
+        timezone = domain.get_default_timezone()
     except pytz.UnknownTimeZoneError:
         timezone = utc
 
-    # check if user has already synced today (in local timezone). Indicators only change daily.
-    last_sync_utc = last_sync.date if last_sync.date.tzinfo else utc.localize(last_sync.date)
-    last_sync_local = timezone.normalize(last_sync_utc.astimezone(timezone))
+    _assert = soft_assert(to=['droberts' + '@' + 'dimagi.com'])
+
+    last_sync_utc = last_sync.date
+
+    if not _assert(last_sync_utc.tzinfo is None,
+                   'last_sync.date should be an offset-naive dt'):
+        last_sync_utc = UserTime(last_sync_utc).server_time().done()
```

Comment:
Well, I think it wouldn't happen, and tracing the code it looks like it never happens, but the code I'm replacing had a provision for it having a tzinfo, so this is my no-risk way of asserting it doesn't.